tests/main.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect, _electron as electron } from '@playwright/test';
+import { rm } from 'node:fs/promises';
+import { join } from 'node:path';
+
+
+test.afterAll(async ({}, testInfo) => {
+  rm(join(testInfo.outputDir, 'drupal'), { recursive:true } );
+});
+
+test('happy path test', async ({}, testInfo) => {
```

Comment:
Nitpick: "test happy path test" is redundant.

```suggestion
test('happy path', async ({}, testInfo) => {
```